README.md

```diff
@@ -54,7 +54,49 @@ env, err := cel.NewEnv(
 
 That's it, the environment is ready to be use for parsing and type-checking.
 CEL supports all the usual primitive types in addition to lists, maps, as well
-as first-class support for JSON and Protocol Buffers.
+as first-class support for JSON and Protocol Buffers. More details on adding
+Protocol Buffers are below.
+
+#### Support for Protocol Buffers
+
+There are several methods that can be used to add a protocol buffer message to
+the environment.
+
+A single protocol buffer message type can be defined like so:
+```go
+import(
+    "github.com/google/cel-go/cel"
+    "github.com/google/cel-go/checker/decls"
+
+    pb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
+)
+
+	env, err := cel.NewEnv(
+		cel.Container("google.api.expr.v1alpha1"),
```

Comment:
Note, the `Container()` isn't necessary unless you intend to reference the type or its enums by relative names in the CEL expression. Consider removing the `Container` unless you intend to add a later examples that shows how the expression changes when you do.